.lintstagedrc.json

```
@@ -0,0 +1,12 @@
+{
+  "*.ts": [
+    "npx eslint --max-warnings=0 --fix",
+    "npx prettier --write",
+    "npm test"
+  ],
+  "*.tsx": [
+    "npx eslint --max-warnings=0 --fix",
+    "npx prettier --write",
+    "npm test"
+  ]
+}
```

Podemos simplificar dessa forma (e também executar o novo commando `test:staged`):

```suggestion
{
  "*.{ts,tsx}": [
    "npx eslint --max-warnings=0 --fix",
    "npx prettier --write",
    "npm test:staged"
  ]
}
```

---

E ao usar o `lint-staged` não precisamos adicionar `npx` no início do script, já que ele (`lint-staged`) é executado como um script do nosso projeto, tendo, assim, acesso à todas as dependência sem necessidade do `npx`.

---

Bom trabalho, ficou conciso dessa forma.

Sobre o npx eu não sabia!! 🤣

Commit realizado!